### DISCUSSION TITLE
Looking for OpenAPI with PHP8 atrributes, symfony, NelmioApiDocBundle learning resources

### DISCUSSION THREAD

**Discussion by darius-v** [OAI/OpenAPI-Specification#4099] [Q&A]
Hello, having trouble to find learning resources about how to use PHP8 attributes with openAPI, symfony. Using [Nelmio](https://symfony.com/bundles/NelmioApiDocBundle/current/index.html) but really getting often various problems which I am not able to find answers to. Have read symfony Nelmio bundle documentation, some other documentations but I can't remember there being talked about problems I have.
For example recently I was strugling to overwrite property of the documenation for specific class field. It has some documentation, class is reused. In one place I wanted to change the documentation for that field without duplicating everything. Was getting so much errors and only when finnally one guy helped in symfony slack, I was able to solve. If I had seen this problem in tutorial, I could have remembered and solved quickly.

Tried searching in udemy but I find only general openAPI tutorials or general PHP, general how to make API. But not a combination of php, openapi, symfony, or at least without symfony. But probably since I use NelmioApiDocBundle, it has some effect also.

**Comment by handrews** [comment]
@darius-v I'm afraid this is the repository for working on the specification itself, and we are not able to offer advice on using specific tools.  You'll need to check with the tool vendor or other PHP-specific resources.  You could also check on the OpenAPI Slack, which you can get to thorugh the "Participate" menu on openapis.org
